recipes/mapbox_earcut/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "mapbox_earcut" %}
+{% set version = "0.12.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 50d995673ac9ce8cb9abb7ab64b709d611c1d27557907e00b631b5272345c453
+  patches:
+    - 0001-removed-setup_requires.patch
+
+build:
+  number: 1
```

This should start from 0 and it only needs to be incremented once packages have been uploaded to anaconda.org:
```suggestion
  number: 0
```